packages/design-system/src/stories/Status.block.tsx

```diff
@@ -0,0 +1,9 @@
+import statuses from './status.json';
+import { Statuses, ComponentStatuses } from '@talend/storybook-docs';
```

Comment:
@talend/storybook-docs should be added as dependency

Reply:
I will use a joker here to fix that in a later PR.
We must remove the DS from all our docs blocks and just use plain react or svg things.